### PR TITLE
v1.72.0 -- Updated orderCard component for Your Favourites rollout

### PR DESCRIPTION
**Pull request by InsungMulumba**
## Changed
 - Order card updated for Your Favourites rollout

image

- [x] This PR has been checked with regard to our brand guidelines

## Browsers Tested

- [x] Chrome
- [ ] Edge
- [ ] Internet Explorer 11
- [ ] Mobile (i.e. iPhone/Android - please list device)

### CODE REVIEW

File: src/scss/components/_media-element.scss [hunk 1]
```diff
@@ -159,7 +159,7 @@ $mediaElement-spacing           : spacing(x2);
         text-decoration: none;
         top: $mediaElement-spacing;
 
-        @include media('>=narrow') {
+        @include media('<mid') {
```
Comment (ashleynolan) [inline src/scss/components/_media-element.scss]: Will this change affect any other components? (Might be used on menu for instance)

Comment (smilburn1990) [reply src/scss/components/_media-element.scss]: @ashleynolan - this is a custom media query I made for the last release but asked Insung to change it for me. Sorry for the confusion 👍

File: src/scss/components/optional/_order-card.scss [hunk 2]
```diff
@@ -103,6 +103,10 @@ $previousOrder-hoverBoxShadow: 0 2px 8px 0 rgba(0, 0, 0, 0.16), -2px -2px 4px 0
         @include font-size(base--scaleUp, false);
         font-weight: $font-weight-bold;
         line-height: line-height(16px, 24px);
+        font-family: $font-family-headings;
+        font-weight: $font-weight-base;
```
Comment (DamianMullins) [inline src/scss/components/optional/_order-card.scss]: This rule is duplicated and is causing a lint error.

File: src/scss/components/optional/_order-card.scss [hunk 3]
```diff
@@ -103,6 +103,10 @@ $previousOrder-hoverBoxShadow: 0 2px 8px 0 rgba(0, 0, 0, 0.16), -2px -2px 4px 0
...
+        @include font-size(22px);
+        padding-top: 2px;
```
Comment (DamianMullins) [inline src/scss/components/optional/_order-card.scss]: Can you add this style to the `padding` rule below. [It's causing a lint error](https://travis-ci.org/justeat/fozzie/jobs/605972644#L244).

**Comment by InsungMulumba** [comment]
bump

File: src/scss/components/optional/_order-card.scss [hunk 4]
```diff
@@ -101,12 +101,14 @@ $previousOrder-hoverBoxShadow: 0 2px 8px 0 rgba(0, 0, 0, 0.16), -2px -2px 4px 0
 
     .c-orderCard-order {
         @include font-size(base--scaleUp, false);
-        font-weight: $font-weight-bold;
         line-height: line-height(16px, 24px);
+        font-family: $font-family-headings;
+        font-weight: $font-weight-base;
+        @include font-size(22px);
```
Comment (kevinrodrigues) [inline src/scss/components/optional/_order-card.scss]: This one is duplicated too?

File: src/scss/components/optional/_order-card.scss [hunk 5]
```diff
@@ -155,4 +157,36 @@ $previousOrder-hoverBoxShadow: 0 2px 8px 0 rgba(0, 0, 0, 0.16), -2px -2px 4px 0
         padding: 0 spacing(x2);
         @include font-size(base--scaleUp, false);
     }
-}
+
+    .c-orderCard-cuisines {
+        color: $grey--mid;
+        margin-top: 0;
+        text-transform: capitalize;
+        font-size: spacing(x2);
```
Comment (kevinrodrigues) [inline src/scss/components/optional/_order-card.scss]: The value here is `spacing` should be a font size value right?

File: CHANGELOG.md [hunk 6]
```diff
@@ -3,6 +3,16 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
+v1.72.0
+------------------------------
+*November 1, 2019*
+
+### Updated
```
Comment (kevinrodrigues) [inline CHANGELOG.md]: There are two `Updated` sections.. but on that note, `Updated` isn't part of the changelog naming convention: https://keepachangelog.com/en/1.0.0/#how

Comment (ashleynolan) [reply CHANGELOG.md]: Should be `Changed` I think?

File: src/scss/components/optional/_order-card.scss (hunk 5 above)
Comment (ashleynolan) [reply src/scss/components/optional/_order-card.scss]: If it should be 12px, can just use `@include font-size(small);`

**Pull request merged**